package.json

```diff
@@ -24,6 +24,7 @@
     "@yarnpkg/plugin-essentials": "^3.2.0-rc.8",
     "@yarnpkg/plugin-git": "^2.6.0-rc.10",
     "clipanion": "^3.2.0-rc.11",
+    "markdown-table": "^3.0.2",
```

Comment:
I'd prefer not to pull an extra dependency, as we could reuse the `DependencyTable` class with some modifications to use plain text formatting and an option to surround each row with pipe symbols.